web/libs/editor/src/components/App/App.jsx

```diff
@@ -260,16 +268,23 @@ class App extends Component {
           >
             {outlinerEnabled ? (
               newUIEnabled ? (
-                <SideTabsPanels
-                  panelsHidden={viewingAll}
-                  currentEntity={as.selectedHistory ?? as.selected}
-                  regions={as.selected.regionStore}
-                  showComments={store.hasInterface("annotations:comments")}
-                  focusTab={store.commentStore.tooltipMessage ? "comments" : null}
-                >
-                  {mainContent}
-                  {store.hasInterface("topbar") && <BottomBar store={store} />}
-                </SideTabsPanels>
+                shouldShowPanels ? (
```

Comment:
should this be applied for situations where `newUIEnabled` is false? IE are we supporting this for situations where FF_DEV_3873 is turned off?

Reply:
Thank you! I have checked we have this flag switched on not everywhere.
So I've changed the code and now it might be displayed right for both cases if `FF_DEV_1170` is on.